lib/v2/huoxian/radar.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+    'huoxian.cn': {
+        _name: '火线',
+        zone: [
+            {
+                title: '火线',
+                docs: 'https://p7e4-musical-umbrella-wgv6vrww59xh59xj-8080.preview.app.github.dev/bbs.html#huo-xian',
```

Comment:
```suggestion
                docs: 'https://docs.rsshub.app/bbs.html#huo-xian',
```
https://github.com/DIYgod/RSSHub/actions/runs/5551022391/jobs/10136706593?pr=12784#step:6:19